pkg/convert/tf_test.go

```diff
@@ -65,3 +65,13 @@ func TestProjectListToSingleton(t *testing.T) {
 		})
 	}
 }
+
+func TestResolveLatestProviderVersion(t *testing.T) {
+	t.Parallel()
+	name := impliedProvider("tfe_organization")
+	provider, err := resolveRequiredProvider(name)
+	assert.NoError(t, err)
+	assert.Equal(t, "tfe", provider.Name)
+	assert.Equal(t, "registry.terraform.io/hashicorp/tfe", provider.Source)
+	assert.Equal(t, "~> 0.70.0", provider.Requirement.Required.String())
```

Comment:
Unfortunately this test fails if the `tfe` provider updates their version but not sure how to test this piece of code otherwise (without tons of indirection)

Reply:
Leave a comment to that effect